src/pages/Login.tsx

```diff
@@ -13,7 +13,8 @@ export default function Login() {
     setIsLoading(true);
 
     // Simulate API call
-    await new Promise((resolve) => setTimeout(resolve, 1000));
+    // await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Login api to be added here with POST, setup

Reply:
Indeed, there will be issues with the appointment section.